src/main.rs

```diff
@@ -7,7 +7,7 @@ mod ui;
 
 fn main() {
   let matches = App::new("CLI Drink")
-    .version("1.0.0")
+    .version("0.1.1")
```

Comment:
I wonder if there's a macro for this?

Reply:
bump yourself to clap 3: https://docs.rs/clap/latest/clap/macro.command.html

Reply:
@Mstrodl I'm gonna let you handle that ;)